package.json

```diff
@@ -21,57 +22,65 @@
     "xml-js": "^1.6.11"
   },
   "devDependencies": {
-    "@commitlint/cli": "8.3.5",
-    "@commitlint/config-conventional": "8.3.4",
-    "@types/classnames": "2.2.9",
-    "@types/node": "13.7.6",
-    "@typescript-eslint/eslint-plugin": "2.21.0",
-    "@typescript-eslint/parser": "2.21.0",
-    "conventional-github-releaser": "^3.1.3",
-    "husky": "3.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
```

Comment:
we have to remove `@playkit-js-contrib` dependency. All methods should be moved in utils.